.gitlab-ci.yml

```diff
@@ -16,9 +16,9 @@ before_script:
     - export CORGI_TEST_OSBS_HOST1
     - export CORGI_TEST_OSBS_HOST2
     - export CORGI_TEST_OSBS_HOST3
-    - export CORGI_TEST_PKGS_HOST
     - export CORGI_TEST_REGISTRY_URL
-    # Not used in tests directly, but needed for tests to pass
+  # Not used in tests directly, but needed for tests to pass
+    - export CORGI_LOOKASIDE_CACHE_URL
     - export CORGI_APP_STREAMS_LIFE_CYCLE_URL
```

Comment:
This is not quite right - this variable is used directly in the tests so should be part of the group above. I'm fine with the rename though if that's more clear to people.

In general all vars with CORGI_TEST_ are needed in Gitlab CI as well as locally / in OpenShift (value is used directly in tests), and the ones in this section are used only locally / in OpenShift (not used in Gitlab and removing them will make operations / local dev fail, but tests will continue to pass).

Reply:
I guess the main reason for renaming this env var was that it's used in `config/settings/base.py`. Also the old name didn't give any indication as to what the variable represented. I guess you're right though and it should be put in the group above this comment as it's also used directly in tests.